main_routers/system_router.py

```
@@ -813,53 +826,133 @@ async def proactive_chat(request: Request):
+                                    for title in titles:
+                                        logger.info(f"  - {title}")
+                            else:
+                                posts = video_data.get('posts', [])
+                                titles = [post.get('title', '') for post in posts[:5]]
+                                if titles:
+                                    logger.info(f"[{lanlan_name}] 成功获取Reddit热门帖子:")
+                                    for title in titles:
+                                        logger.info(f"  - {title}")
+                    
+                if content_type is None or content_type not in ('news', 'video'):
+                    # 获取全部内容（首页推荐）
```

Ruff (0.15.0)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)